SwiftUIIntegration/ContentView.swift

```diff
@@ -0,0 +1,191 @@
+//
+//  ContentView.swift
+//  SwiftUIIntegration
+//
+//  Copyright © 2025 Zendesk. All rights reserved.
+//
+
+import SwiftUI
+import ZendeskSDK
+import ZendeskSDKMessaging
+import OSLog
+
+/// The main content view for the Zedesk SDK Demo App.
+struct ContentView: View {
+
+    @State private var channelKey: String
+    @State private var jwt: String
+    @State private var currentUser: ZendeskSDK.ZendeskUser?
+    @State private var isInitialized: Bool = false {
+        didSet {
+            if isInitialized, !jwt.isEmpty { loginUser()}
+        }
+    }
+    @State private var isLoggedIn: Bool = false {
+        didSet {
+            if !isLoggedIn { currentUser = nil }
+        }
+    }
+    @State private var pageViewTitle: String = ""
+    @State private var pageViewUrl: String = ""
+
+    private let logger = Logger(subsystem: Constants.bundleIdentifier, category: Constants.category)
+
+    /// Initialize the ContentView with the stored channel key and JWT token.
+    init() {
+        channelKey = UserDefaults.standard.string(forKey: Constants.channelKey) ?? ""
+        jwt = UserDefaults.standard.string(forKey: Constants.jwtKey) ?? ""
+    }
+
+    var body: some View {
+        NavigationView {
+            List {
+                Section(header: InfoBannerView.zendeskInitialization) {
+                    InitializationItem(channelKey: $channelKey, initialize: {
+                        initializeZendeskSDK()
+                    }, invalidate: { clearStorage in
+                        invalidateZendeskSDK(clearStorage: clearStorage)
+                    })
+                    NavigationLink(destination: MessagingView()) {
+                        InfoBannerView.showMessaging
+                    }
+                    .disabled(!isInitialized)
+                    .opacity(isInitialized ? 1 : 0.5)
+                }
+                Section(header: InfoBannerView.authentication) {
+                    AuthenticationItem(jwt: $jwt, login: {
+                        loginUser()
+                    }, logout: {
+                        logoutUser()
+                    })
+                    if let currentUser {
+                        Text("User authenticated: \(currentUser.id)")
+                    } else {
+                        Text("No user authenticated")
+                    }
+                }
+                .disabled(!isInitialized)
+                .opacity(isInitialized ? 1 : 0.5)
+                Section(header: InfoBannerView.pageView) {
+                    ClearableTextField(placeholder: "Page view title", text: $pageViewTitle)
+                    ClearableTextField(placeholder: "Page view URL", text: $pageViewUrl)
+                    Button(action: {
```

Comment:
Not necessary, but there is simpler API:
```suggestion
                    Button {
```